insights/parsers/gcp_license_codes.py

```diff
@@ -0,0 +1,60 @@
+"""
+GCPLicenseCodes
+===============
+
+This parser reads the output of a command
+``curl -H "Metadata-Flavor: Google" "http://metadata.google.internal/computeMetadata/v1/instance/licenses/?recursive=True"``,
+which is used to check whether the google cloud instance is a licensed marketplace instance.
+
+For more details, See: https://cloud.google.com/compute/docs/reference/rest/v1/images/get#body.Image.FIELDS.license_code
+
+"""
+import json
+
+from insights.parsers import SkipException, ParseException
+from insights import parser, CommandParser
+from insights.specs import Specs
+
+
+@parser(Specs.gcp_license_codes)
+class GCPLicenseCodes(CommandParser):
+    """
+    Class for parsing the GCP License Codes returned by command
+    ``curl -H "Metadata-Flavor: Google" "http://metadata.google.internal/computeMetadata/v1/instance/licenses/?recursive=True"``,
+
+
+    Typical Output of this command is::
+
+        [{"id": "601259152637613565"}]
+
+    Raises:
+        SkipException: When content is empty or no parse-able content.
+
```

Comment:
Since you are making changes could you add ParseException to this section of the doc since you raises it as well? Thx.

Reply:
oh snap. yep